addons/iap/static/src/action_buttons_widget/action_buttons_widget.js

```diff
@@ -12,11 +12,7 @@ class IAPActionButtonsWidget extends Component {
     }
 
     async onViewServicesClicked() {
-        const url = await this.orm.silent.call("iap.account", "get_account_url");
```

Comment:
I think this was the last reference to the `get_account_url` function, its code can be removed.